src/plugins/discover/public/components/discover_grid/discover_grid_columns.test.tsx

```diff
@@ -281,20 +281,24 @@ describe('Discover grid columns', function () {
             [Function],
             [Function],
           ],
-          "display": <React.Fragment>
-            timestamp
-             
-            <EuiIconTip
-              aria-label="Primary time field."
+          "display": <div
+            aria-label="Primary time field."
+          >
+            <EuiToolTip
               content="This field represents the time that events occurred."
-              iconProps={
-                Object {
-                  "tabIndex": -1,
-                }
-              }
-              type="clock"
-            />
-          </React.Fragment>,
+              delay="regular"
+              display="inlineBlock"
+              position="bottom"
+            >
+              <React.Fragment>
+                timestamp
```

Comment:
I think while it makes sense to show the tooltip also when hovering the field name, it breaks the functionals, so I'd suggest to use the tooltip only with the clock icon.

Reply:
It breaks the functional tests when attempting to click a menu button that's obscured by the tooltip, like this:
![tooltip_obscuring_menu](https://user-images.githubusercontent.com/25592674/173921486-0ef0334a-87f3-4c09-a8cb-d61b6a446616.png)

Reply:
Ah lmfao. I'd argue adding a line to the functional test moving the cursor down or up 10px would be worth the increased UX, but no worries, it's not a super strongly held opinion/blocker. Feel free to just move the wrapping `<EuiTooltip>` to just over the `<EuiIcon>` 🤷

Reply:
Fantastic!!